swagger_to_sdk_config.json

```diff
@@ -460,10 +460,10 @@
       "output_dir": "azure-mgmt-web/azure/mgmt/web/"
     },
     "batch": {
-      "swagger": "batch/2017-01-01.4.0/swagger/BatchService.json",
+      "swagger": "batch/2017-05-01.5.0/swagger/BatchService.json",
       "autorest_options": {
         "Namespace": "azure.batch",
-        "PackageVersion": "2.0.1"
+        "PackageVersion": "3.0.0"
```

Comment:
could you add "build_dir": "azure-batch" just after "output_dir" and restart the generate script?